src/main/java/androidx/view/View.kt

```diff
@@ -84,6 +85,17 @@ inline fun View.doOnPreDraw(crossinline action: (view: View) -> Unit) {
     })
 }
 
+/**
+ * Sends {@link AccessibilityEvent} of type {@link AccessibilityEvent#TYPE_ANNOUNCEMENT}.
```

Comment:
Make sure you use Dokka instead of JavaDocs.

You will need these imports:
```
import android.view.accessibility.AccessibilityEvent
import android.view.accessibility.AccessibilityEvent.TYPE_ANNOUNCEMENT
```

And change `Sends {@link AccessibilityEvent} of type {@link AccessibilityEvent#TYPE_ANNOUNCEMENT}.` to `Sends [AccessibilityEvent] of type [AccessibilityEvent.TYPE_ANNOUNCEMENT].`.

Reply:
Thanks!